fractal-terrain-generation/builder.js

```diff
@@ -0,0 +1,67 @@
+const replace = require('replace-in-file');
+const fse = require('fs-extra');
+const path = require('path');
+
+const YMLUtil = require('binaris/lib/binarisYML');
+
+const { getAccountId, getAPIKey, getRealm } = require('binaris/lib/userConf');
+const { forceRealm } = require('binaris/sdk');
+
+const backendYAMLPath = path.join(__dirname, 'fractal_backend');
+const servingYAMLPath = path.join(__dirname, 'dist');
+
+async function getServingFuncName() {
+  const binarisConf = await YMLUtil.loadBinarisConf(servingYAMLPath);
+  return YMLUtil.getFuncName(binarisConf);
+}
+
+async function getBackendFuncName() {
+  const binarisConf = await YMLUtil.loadBinarisConf(backendYAMLPath);
+  return YMLUtil.getFuncName(binarisConf);
+}
+
+async function getPublicPath(accountID, endpoint) {
+  process.env.BINARIS_INVOKE_ENDPOINT = endpoint;
```

Comment:
Is overriding `process.env.BINARIS_INVOKE_ENDPOINT` before storing it in `savedEndpoint` a bug?

Also, this assumes that `binaris/sdk/url` behaves in a certain way - if you need a specific functionality from the SDK maybe this function should be in the SDK?

Reply:
1. It's not a bug. Our SDK relies on Environment variables and cannot be overridden with passed values. I definitely agree this could prompt a change but that could entail a lengthy discussion. How about we do that in parallel?
2. It does assume but the alternative is exposing bloat to the user. For now I'm ok with this hack until we fix the underlying issues

Reply:
Do we still need this whole function with the sandbox shutdown?